flow-model-generator/modules/flow-model-generator-ls-extension/src/main/java/io/ballerina/flowmodelgenerator/extension/response/FunctionCallTemplateResponse.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com)
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+ *  of any KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.flowmodelgenerator.extension.response;
+
+/**
+ * Response class for function call template.
+ *
+ * @since 2.0.0
+ */
+public class FunctionCallTemplateResponse extends AbstractFlowModelResponse {
+    private String template;
+
+    public void setTemplate(String template) {
+        this.template = template;
+    }
+
+    public String template() {
+        return template;
+    }
+}
```

Comment:
Is there a reason why we can't use a record in here instead?

Reply:
We are extending the `AbstractFlowModelResponse`, but records don't support inheritance

Reply:
didnt notice that this is an extended class. my bad